camera/camera_utils.cc

```diff
@@ -0,0 +1,51 @@
+#include "camera/camera_utils.hh"
```

Comment:
Avoid files named `_utils` and `_helpers`, as they end up becoming `_misc_everything` --> prefer for example `initialize_camera.hh` or `camera_interface.hh`